spec/fixtures/api/singleton-userdata/main.js

```diff
@@ -0,0 +1,12 @@
+const { app } = require('electron');
+const fs = require('fs');
+const path = require('path');
+
+// non-existing user data folder should not break requestSingleInstanceLock()
+// ref: https://github.com/electron/electron/issues/33547
+const userDataFolder = path.join(app.getPath('home'), 'singleton-userdata');
```

Comment:
Can you also prefix this with `electron-test-` to align with https://github.com/electron/electron/pull/33511